test/graph.jl

```diff
@@ -53,6 +53,30 @@ using DataStructures
         @test node_attributes == correct_attributes
     end
 
+    @testset "Extracting coordinates and minimum bounding rectangles" begin
+        # file should have a .json or .shp extension
+        table = GerryChain.read_table(square_shp_filepath)
+        correct_coords = [
+            [[[[0.0, 0.0], [0.0, 1.0], [1.0, 1.0], [1.0, 0.0], [0.0, 0.0]]]],
+            [[[[0.0, 1.0], [0.0, 2.0], [1.0, 2.0], [1.0, 1.0], [0.0, 1.0]]]],
+            [[[[1.0, 0.0], [1.0, 1.0], [2.0, 1.0], [2.0, 0.0], [1.0, 0.0]]]],
+            [[[[1.0, 1.0], [1.0, 2.0], [2.0, 2.0], [2.0, 1.0], [1.0, 1.0]]]]
+        ]
+        # extract coordinates
+        coords = GerryChain.get_node_coordinates.(table)
```

Comment:
I like the use of dotted functions, I am going to start using them more!

Reply:
yes! they are super nifty and cleaner than using `map` or `[ f(x) for x in X ]`. i support it 👍